src/main/java/com/github/krzysiekjodlowski/ox/OxGameEventBus.java

```diff
@@ -0,0 +1,32 @@
+package com.github.krzysiekjodlowski.ox;
+
+import com.github.krzysiekjodlowski.ox.board.BoardValidator;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * ${@inheritDoc}.
+ *
+ * @author Krzysztof Jodlowski
+ */
+class OxGameEventBus implements EventBus {
+
+  private final List<Subscriber> subscribers = new ArrayList<>();
```

Comment:
You could use LinkedList here

Reply:
Especially that I'm using forEach here - changed!